home/tests/tests.py

```diff
@@ -98,3 +102,35 @@ def test_media_block_translation_of_bengali_language(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, f"উপরের ভিডিও দেখা না গেলে <a href=\"{self.en_article.body[0].value.url}\" download> এর পরিবর্তে এটা </a> ডাউনলোড করুন", count=1)
         self.assertContains(response, f"উপরের অডিও শুনতে না পেলে <a href=\"{self.en_article.body[1].value.url}\" download> এর পরিবর্তে এটা </a> ডাউনলোড করুন", count=1)
+
+
+class FlatMenuTest(TestCase):
```

Comment:
This may be one of those cases when a test is not necessary. I don't think this is providing much value to us, and the consequences of not catching a regression are not particularly difficult to deal with - the management command can be applied, or a manual workaround via the CMS UI. My opinion is that we should remove it.

Reply:
Okay, we can go for a manual workaround via the CMS UI.